packages/pn-pa-webapp/src/pages/components/NewNotification/Attachments.tsx

```diff
@@ -177,7 +185,27 @@ const Attachments = ({ onConfirm, onPreviousStep, attachmentsData, forwardedRef
     validationSchema,
     validateOnMount: true,
     onSubmit: (values) => {
-      if (formik.isValid) {
+      if (!IS_PAYMENT_ENABLED) {
```

Comment:
This sound strange to me.
I would expect this workflow:
- form is valid (formik.isValid)?
  - No -> do nothing
  - Yes -> continue to the next point
- isCompleted and !IS_PAYMENT_ENABLED?
  - No -> do the two dispatches
  - Yes -> call onConfirm and don't call two dispatches